src/test/java/Chatbot/ParserTest.java

```diff
@@ -0,0 +1,50 @@
+package Chatbot;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
```

Comment:
There were ample testcases for the different possible inputs, well done! Perhaps you could try some improperly formatted inputs, eg
```java
"event project meeting /from 2024-09-01 09:00 /to 2024-09-0110:00"
"event project meeting /from  /to 2024-09-01 10:00"
"event /from 2024-09-01 09:00 /to 2024-09-01 10:00"
```